auth-api/src/auth_api/services/affidavit.py

```diff
@@ -90,8 +92,34 @@ def create_affidavit(affidavit_info: Dict):
 
         affidavit_model.save()
 
+        if trigger_task_update:
+            Affidavit._modify_task(user)
+
         return Affidavit(affidavit_model)
 
+    @staticmethod
+    def _modify_task(user):
+        # find users org. ideally only one org
```

Comment:
Can you add more comments here to make it clear what's happening here. Shouldn't the old task be marked as completed, since you are creating a new task. Can't find the code piece where it marks the old task as complete.